frontend/app/profile/ProfileForm.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import {StyleSheet, View} from 'react-native';
+import {WHITE} from '../colors';
+import TextInputField from '../shared/TextInputField';
+
+type ProfileFormProps = {
+  username: string;
+  setUsername: (text: string) => void;
+  password: string;
+  setPassword: (text: string) => void;
+};
+
+export default function ProfileForm({
+  username,
+  setUsername,
+  password,
+  setPassword,
+}: ProfileFormProps) {
+  return (
+    <View style={styles.container}>
+      <TextInputField
+        title="Username"
+        onChangeText={setUsername}
+        value={username}
+      />
+      <TextInputField
+        title="Password"
```

Comment:
Can you change the TextInputField component, so that it allows to set the type of input? E.g. password, so that it is not shown in plain text